src/Common/DuplicateSubmitButton.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace ipl\Web\Common;
+
+use ipl\Html\Attribute;
+use ipl\Html\Contract\FormSubmitElement;
+use ipl\Html\FormElement\SubmitElement;
+
+/**
+ * Provides the `duplicateSubmitButton` method, used to force the primary submit element in forms
+ */
+trait DuplicateSubmitButton
```

Comment:
In my opinion, this trait is no longer useful. It is called `DuplicateSubmitButton`, although its intent is to always clone the primary submit button, and it also hard codes the CSS class to use. I opt to remove the trait and just clone directly in the form. Please keep that in the method as it is easier to test.